frontend/src/components/Topbar/Topbar.jsx

```
@@ -0,0 +1,33 @@
+        <FaStar />
+        <span>Add to favorites</span>
+      </button>
+      <div className="ml-auto flex items-center gap-4">
+        <div className="relative">
+          <input
+            className="bg-[#23272f] border border-[#333] rounded px-3 py-1 text-sm text-white focus:outline-none"
+            placeholder="Search..."
+          />
+          <FaSearch className="absolute right-2 top-2 text-gray-500 text-xs" />
+        </div>
+        <button className="bg-[#b388ff] text-white px-3 py-1 rounded text-sm font-semibold ml-2">
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)